routes/user.js

```diff
@@ -577,6 +578,44 @@ router.post("/avatar", async function (req, res) {
     }
 });
 
+router.post("/birthday", async function (req, res){
+    res.setHeader("Content-Type", "application/json");
+    try{
+        let userId = await routeUtils.verifyLoggedIn(req);
+        let prop = String(req.body.prop);
+        var itemsOwned = await redis.getUserItemsOwned(userId);
+        var perm = "changeBday";

```

Comment:
maybe skip the permission check?

I reloaded backend + frontend, still got this permission error

how did you add permission for your user?
the concern is that all existing users won't have the bday permission

![image](https://user-images.githubusercontent.com/24848927/226355447-ffe73281-e431-4bd7-9db0-de12cd064f3b.png)